server/api/index.js

```diff
@@ -1,31 +1,26 @@
-const Shopify = require("shopify-api-node");
-const express = require("express");
-const axios = require("axios");
+const Shopify = require('shopify-api-node');
 
 const { SHOPIFY_SHOP_NAME, SHOPIFY_API_KEY, SHOPIFY_PASSWORD } = process.env;
-const router = express.Router();
 
 const shopify = new Shopify({
-  shopName: SHOPIFY_SHOP_NAME,
-  apiKey: SHOPIFY_API_KEY,
-  password: SHOPIFY_PASSWORD
+	shopName: SHOPIFY_SHOP_NAME,
+	apiKey: SHOPIFY_API_KEY,
+	password: SHOPIFY_PASSWORD,
 });
 
-shopify.on("callLimits", limits => console.log(limits, "test2"));
+const create = param => {
+	shopify.order
+		.create(param)
+		.then(data => console.log(data))
+		.catch(err => console.error(err));
+};
 
-router.post("/transaction", async (req, res) => {
-  console.log(req.body);
-  const url = "https://weglimpse.co/admin/api/2019-10/reports.json";
-  try {
-    const { data } = await axios.get(url);
-
-    if (data) {
-      res.sendStatus(200).json(data);
-    }
-  } catch (error) {
-    console.log(error);
-    res.sendStatus(500).json(error);
-  }
-});
-
-module.exports = router;
+shopify.order
+	.list({ financial_status: 'paid' })
+	.then(orders => {
+		const totalPrice = orders.reduce((acc, value, index) => {
+			return acc + Number(value.total_price);
+		}, 0);
+		create({ totalPrice });
```

Comment:
We don't want to create a new order! You can just replace this with a console log here for now, then later we will implement sending this data to the client.